services/src/main/java/org/keycloak/services/resources/admin/RealmAdminResource.java

```diff
@@ -1074,8 +1075,22 @@ public RealmRepresentation partialExport(@QueryParam("exportGroupsAndRoles") Boo
         // this means that if clients is true but groups/roles is false the service account is exported without roles
         // the other option is just include service accounts if clientsExported && groupsAndRolesExported
         ExportOptions options = new ExportOptions(false, clientsExported, groupsAndRolesExported, clientsExported);
-        RealmRepresentation rep = ExportUtils.exportRealm(session, realm, options, false);
-        return stripForExport(session, rep);
+
+        ExportProvider exportProvider = session.getProvider(ExportProvider.class);
```

Comment:
The latest commit moves adds the functionality to the ExportImportManager. To support the JSON export, the necessary code from ExportUtils has been duplicated in MapExportImportManager, as ExportUtils is not accessible from the map storage.

If we don't want to duplicate the code, it could be part of `ModelToRepresentation`.

A future MapExportImportManager could allow different map storages with different implementations.

Reply:
We will only need JSON import in the map storage, we can remove the export.

Reply:
Ok, removed it from the code of MapExportImportManager. 

Once I tried to move over the interface ExportImportManager to org.keycloak.exportimport, it clashes with a class of the same name in the services module in that package. That class will need to be refactored eventually in relation to the new store, and I then postponed the move until we do that. 

Now everything should compile, and manual test did run OK for the legacy store, and I'll now look at the tests.